index.html

```diff
@@ -1209,46 +1215,21 @@ <h3>Algorithm</h3>
                 <li>Otherwise, initialize <var>result</var> as a
                   newly-initialized <var>active context</var>,
                   <span class="changed">setting <a>previous context</a> in <var>result</var>
-                    to the previous value of <var>result</var> if <var>propagate</var> is <code>false</code></span>.
+                    to the previous value of <var>result</var> if <var>propagate</var> is <code>false</code></span>
+                    and <a>base IRI</a> to the value of the {{JsonLdOptions/base}} option
+                    of the <a href="#the-application-programming-interface" class="sectionRef"></a>,
+                    if provided, otherwise to <var>base URL</var>.
```

Comment:
So "global configuration" (the API parameter) takes precedence over "local configuration" (algorithm parameter) ? This might very well be the way to go, I don't have a clear enough view of the big picture right now... But I'm pointing it out because it is a bit counter-intuitive...

Reply:
In this case, we're going back to the initial state, where _base IRI_ is either explicitly set, or comes from the document location, which is in _base URL_. It's true that this could get it wrong, as if you're deep into remote contexts, _base URL_ will be that of the containing context, rather than the original document.

Perhaps, we should have something accessible from the API which would record the `{{RemoteDocument/documentUrl}}` which could be used instead of _base URL_. Or, perhaps, update the Context Processing algorithm to remember the original _base URL_ passed in that doesn't get changed with each new remote context, just for this purpose.

This is an example of the "ripple effects" that making this change does, which I'm not sure actually clarifies the algorithm.

Reply:
I solved this in dcb7a57c by adding an _original base URL_ component to an active context.